Create 2025.md (#111)
mautic/low-no-code: 2025.md

> # Low- and no-code guide for contributing to Mautic

Author: We don't use camel case in our technical writing, so I'll leave this one as it is. Here's our [writing style guide](https://contribute.mautic.org/education-team/technical-writing-styleguide) :)

Welcome to **Mautic**! We're excited that you're interested in contributing to **Mautic**, an open source marketing automation platform. Contributions come in many forms, and not all of them require coding skills. This guide will show you how to contribute to Mautic without writing a single line of code!

## Table of Contents

> - [Low- and no-code guide for contributing to Mautic](#Low-and-no-code-guide-for-contributing-to-mautic)

Author: Do you mean removing the first item in the ToC because it comes before the ToC? If so I think that's a valid point!

> - [Low- and no-code guide for contributing to Mautic](#Low-and-no-code-guide-for-contributing-to-mautic)

Reviewer: Yes, that's exactly what I mean. 👍🏼

>   - [Table of Contents](#table-of-contents)

Author: Missed this one, fine to remove!

  - [About Mautic](#about-mautic)
  - [What is Hacktoberfest?](#what-is-hacktoberfest)

>   - [Why No-Code Contributions Matter](#why-no-code-contributions-matter)

Author: Thanks for the catch!

    - [1. **Documentation**](#1-documentation)
    - [2. **Translations**](#2-translations)
    - [3. **Community Engagement**](#3-community-engagement)
    - [4. **Design and User Experience (UX)**](#4-design-and-user-experience-ux)
    - [5. **Marketing**](#5-marketing)
    - [6. **Knowledgebase**](#6-knowledgebase)
  - [How to Get Started](#how-to-get-started)
  - [Useful Resources](#useful-resources)

---

## About Mautic

[Mautic](https://www.mautic.org/) is the world’s largest open source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. 

## What is Hacktoberfest?

[Hacktoberfest](https://hacktoberfest.com/) is an annual event encouraging contributions to open source projects. Throughout October, participants are invited to submit contributions to open source repositories like Mautic. Contributions can be code, documentation, design, translations, or community support. This repository was initially created to support the Hacktoberfest event but we're now using it year-round due to the success of Hacktoberfest.

> ## Why low- and no-code contributions matter

Author: Closing this one as we don't use camel case for headings.

Open source projects thrive not only from code contributions but also from the community's efforts in other areas. Low- and no-code contributions help Mautic grow and improve through better documentation, translations, community support, and more. If you're not a developer but want to contribute to open source, there are plenty of ways to get involved!

## Ways to Contribute

There are several ways to contribute to Mautic without writing code:

### 1. **Documentation**
   - **Improve or create new documentation**: Help by editing existing documentation for clarity, or writing new guides, tutorials, and FAQs.

>    - **Review documentation**: Proofread and provide feedback to ensure accuracy and readability.

Author: Thanks! I'm so used to RST my markdown is a bit rusty!

### 2. **Translations**
   - **Translate Mautic**: Assist in translating Mautic’s user interface, documentation, or tutorials into different languages.
   - **Proofread translations**: Review existing translations and help refine them for correctness and localization.

### 3. **Community engagement**
   - **Answer Questions**: Help users in Mautic’s forums, Slack, or other community platforms by answering questions or sharing your experiences.

>    - **Organize or participate in events**: Organize local Mautic meetups, workshops, or help promote Mautic in your community.

Author: I've tweaked so that it's using sentence case, but will accept for the extra spacing :)

   
### 4. **Design and user experience (UX)**
   - **Create or improve UI/UX designs**: Work on wireframes, mockups, and designs that improve the user interface of Mautic.
   - **Usability testing**: Provide feedback on how Mautic can be more user-friendly by testing features and sharing your insights.

### 5. **Marketing**
   - **Promote Mautic**: Spread the word about Mautic through social media, blogs, or video tutorials.
   - **Create content**: Write articles, create videos, or design infographics that explain Mautic’s features and benefits.

### 6. **Knowledgebase**
   - **Contribute to Mautic’s Knowledgebase**: Write helpful articles, how-to guides, or troubleshoot common issues in the [Mautic Knowledgebase](https://kb.mautic.org/).
   - **Update existing articles**: Review and update articles to ensure they reflect the latest Mautic features and best practices.

## How to Get Started

> 1. **Sign up for Hacktoberfest, if you're participating**  
>    - If you're contributing during October, register for Hacktoberfest at [Hacktoberfest website](https://hacktoberfest.com/auth/) using your GitHub account.

Reviewer: **suggestions:**

- This ordered list only has one explanation. We might want to consider to remove the list here.
- We might want to follow the [Adding Element in List best practice](https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists), where we give a space after the ordered list and indent the element four spaces.

```suggestion
1. **Sign up for Hacktoberfest, if you're participating**

    If you're contributing during October, register for Hacktoberfest at [Hacktoberfest website](https://hacktoberfest.com/auth/) using your GitHub account.
```

Author: Thanks!

> 2. **Sign up to contribute to Mautic**  
>    If you're contributing during October, register to participate in Mautic's Hacktoberfest event by filling out the form here: [Mautic Hacktoberfest 2024 Signup](https://m.mautic.org/hacktoberfest-24-signup). Please indicate **how you'd like to contribute** in the form by selecting areas such as documentation, translations, design, marketing, or community support.

Reviewer: **question**: Do we want to change the "Mautic Hacktoberfest 2024 Signup" to 2025? Or we want to wait changing it until the upcoming Hacktoberfest?

```suggestion
2. **Sign up to contribute to Mautic**

    If you're contributing during October, register to participate in Mautic's Hacktoberfest event by filling out the form here: [Mautic Hacktoberfest 2024 Signup](https://m.mautic.org/hacktoberfest-24-signup). Please indicate **how you'd like to contribute** in the form by selecting areas such as documentation, translations, design, marketing, or community support.
```

Author: We haven't made the 2025 signup yet so I just left this at 2024's data (as the form still works)

3. **Join the Mautic community**  
   - Visit the [Mautic Forum](https://forum.mautic.org/) to introduce yourself and engage with the community.
   - Join the [Mautic Slack](https://www.mautic.org/slack) for real-time conversations with contributors and the Mautic team.

> 4. **Pick your contribution type**  
>    Choose the area where you want to contribute, whether it's documentation, translation, marketing, knowledgebase or community support.

Reviewer: **question**: Do we want to capitalize the "knowledgebase" here as in line 127?

```suggestion
4. **Pick your contribution type**

    Choose the area where you want to contribute, whether it's documentation, translation, marketing, knowledgebase or community support.
```

Author: I think it's fine to leave it as-is

> 5. **Find an issue**  
>    Browse through the **[Mautic low- and no-code Issues](https://github.com/orgs/mautic/projects/21/)**. There are specific tags like `documentation`, `design`, or `translation` to help you find no-code contribution opportunities. Use the tabs to navigate between the types of tasks.

Reviewer: ```suggestion
5. **Find an issue**

    Browse through the **[Mautic Low- and No-Code Issues](https://github.com/orgs/mautic/projects/21/)**. There are specific tags like `documentation`, `design`, or `translation` to help you find no-code contribution opportunities. Use the tabs to navigate between the types of tasks.
```

6. **Make Your Contribution**  
   - **Documentation**: Review or improve the documentation at [Mautic Documentation](https://docs.mautic.org/).
   - **Translations**: Get involved by translating at [Mautic Translations](https://www.transifex.com/mautic/mautic/).
   - **Community**: Help in the forums or join conversations on Mautic’s social media platforms.

> 7. **Submit Your Contribution**  
> For documentation, submit your work via a pull request on GitHub - if you're contributing in October make sure to reference Hacktoberfest in your contribution notes so that the team know to add the `hacktoberfest-accepted` label. For other tasks like translations or writing on the knowledgebase, follow the submission process on Transifex or the knowledgebase and then follow the steps below to have your contribution validated and counted. 

Reviewer: **question:** Same question here about capitalizing "knowledgebase".

**suggestions:**
- Give a space after the ordered list and indentation.
- For better understanding, I think period is better to use here than a stripe.
- Add a comma in the 2nd sentence after "If you're contributing in October, ...".

```suggestion
7. **Submit Your Contribution**

    For documentation, submit your work via a pull request on GitHub. If you're contributing in October, make sure to reference Hacktoberfest in your contribution notes so that the team know to add the `hacktoberfest-accepted` label. For other tasks like translations or writing on the knowledgebase, follow the submission process on Transifex or the knowledgebase and then follow the steps below to have your contribution validated and counted. 
```

> For low- and no-code contributions where there is no pull request to make (e.g. marketing, design etc) we've created this file in this repository where we will maintain an activity log, which will allow you to submit a pull request (so that it counts for Hacktoberfest and as a Mautic contribution) each time you complete a task. Please note: some teams work on Jira, and those issues are automatically counted as a contribution when they are completed and assigned to a person, so you don't need to make a PR here for those tasks.

Reviewer: **suggestion:**
- Give a space after the ordered list and indentation.
- Add punctuations for clarity.
- We might want to mention the file name and link it here. If we don't have it yet, referring to it with "a file" then "this file" might be better. Or maybe "a file that typically called `YYYY.md`"?
- Bold the "Please note:" to draw attention.
- I had to read the content in the "please note" a couple of times to understand it. Would below suggestion make it clearer and grammatically correct?

```md
**Please note:** Some teams work on Jira, and those issues are automatically counted as a contribution when they are completed and assigned to a person. If you work with issues on Jira, you don't need to make a PR here for those tasks.
```

---

```suggestion
    For low- and no-code contributions where there is no pull request to make (e.g. marketing, design etc.), we've created a file in this repository where we will maintain an activity log, which will allow you to submit a pull request (so that it counts for Hacktoberfest and as a Mautic contribution) each time you complete a task. **Please note:** Some teams work on Jira, and those issues are automatically counted as a contribution when they are completed and assigned to a person, so you don't need to make a PR here for those tasks.
```

Author: I figured not mentioning the name makes it easier to re-use but if we want to mention the file that's fine with me also.

To make a PR to have your contribution counted, follow these steps:

1. Log in to GitHub with your GitHub account
2. Click the Fork button at the top right of this repository
3. Edit the relevant file for the current year, and add your contribution and a link to the issue at the bottom of the list. Follow any instructions that are written as to the formatting to use
4. Click the button to commit your changes, and make a new branch (ideally call it something relevant to this contribution, so you know what each one is)
5. Go back to the [Mautic low and no-code](https://github.com/mautic/low-no-code) repository
6. Click the button at the top of the page to make a new pull request
7. Add the details about your contribution, and submit the pull request for the team to review
8. Celebrate! 🥳

> Please follow this process for each contribution, so that it counts for Hacktoberfest/for Mautic's contributions. The team will add the label 'hacktoberfest-accepted' for relevant contributions, which will allow you to have your contributions counted.

Reviewer: **suggestions:** 
- Let's remove the word "for" before Mautic's contribution, and use "or" instead of "/" for clarity.
- In line 81, we use backticks for tags. Let's use it here too for consistency.

```suggestion
Please follow this process for each contribution, so that it counts for Hacktoberfest or Mautic's contributions. The team will add the label `hacktoberfest-accepted` for relevant contributions, which will allow you to have your contributions counted.
```

> 8. **Get Your Rewards**  
>    After contributing to four eligible open source repositories during Hacktoberfest, you’ll earn a limited-edition Mautic Hacktoberfest digital badges. For other contributions we're working on unique holopin badges - watch this space!

Reviewer: **typo:** Capitalize "Holopin".

**suggestion:** Add a comma at the 2nd sentence, after "For other contributions, ...".

```suggestion
8. **Get Your Rewards**

    After contributing to four eligible open source repositories during Hacktoberfest, you’ll earn a limited-edition Mautic Hacktoberfest digital badges. For other contributions, we're working on unique Holopin badges - watch this space!
```

## Useful Resources

- [Mautic GitHub Repository](https://github.com/mautic/mautic)
- [Mautic Documentation](https://docs.mautic.org/)
- [Mautic Translations on Transifex](https://www.transifex.com/mautic/mautic/)
- [Mautic Forum](https://forum.mautic.org/)
- [Mautic Slack](https://www.mautic.org/slack)
- [Mautic GitHub Project Tasks](https://github.com/orgs/mautic/projects/21/views/1)
- [Mautic Knowledgebase](https://kb.mautic.org/)
- [Hacktoberfest Website](https://hacktoberfest.com/)

---

Thank you for being part of the Mautic community and for your contributions throughout 2024. Every contribution helps make Mautic better for users around the world!

## 2025 

> ### Hacktoberfest no-code contribution activity log

Reviewer: ```suggestion
### Hacktoberfest No-Code Contribution Activity Log
```

> When linking to your contribution, please write a descriptive title and use the full link of the Jira issue, Knowledgebase article, or other reference. For example [Article on how to contribute to the Knowledgebase](https://kb.mautic.org/article/how-to-contribute-to-the-knowledge-base.html).

Reviewer: **suggestion:** Add a comma after "For example".

```suggestion
When linking to your contribution, please write a descriptive title and use the full link of the Jira issue, Knowledgebase article, or other reference. For example, [Article on how to contribute to the Knowledgebase](https://kb.mautic.org/article/how-to-contribute-to-the-knowledge-base.html).
```
